pkg/api/addtask.go

```diff
@@ -31,13 +33,13 @@ func AddTaskHandler(w http.ResponseWriter, r *http.Request) {
 
 	err = db.CheckDate(&task, now)
 	if err != nil {
-		SendError(w, err.Error())
+		http.Error(w, "can not check date", http.StatusInternalServerError)
```

Comment:
Это же клиент прислал неверные данные
```suggestion
		http.Error(w, "can not check date", http.StatusBadRequest)
```

Reply:
![2025-07-25 14 53 49](https://github.com/user-attachments/assets/ff23f40f-ea47-45f6-a9a3-a87c25222980)

Странно, что при таком вводе проходят все тесты

Reply:
Согласен, в тестах много на самом деле косяков, но мои попытки попросить авторов исправить их проваливались(